src/libstd/process.rs

```diff
@@ -47,6 +47,14 @@ use thread::{self, JoinHandle};
 ///
 /// assert!(ecode.success());
 /// ```
+///
+/// # Note
+///
+/// Take note that there is no implementation of
+/// [`Drop`](../../core/ops/trait.Drop.html) for child processes, so if you
+/// not ensure the `Child` has exited (through `kill`, `wait`, or
+/// `wait_with_output`) then it will continue to run even after the `Child`
```

Comment:
This may want to be rephrased to only indicate that `wait` can be used to guarantee that a child process has gone away. Using `kill` won't actually guarantee it's dead (e.g. it may still take awhile) and the `wait_with_output` method just delegates to `wait` internally.

There also may be a missing "do" in the phrase "so if you not ensure"